src/Microsoft.Security.Utilities.Core/Microsoft.Security.Utilities.Core.csproj

```diff
@@ -43,6 +43,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Web" />
```

Comment:
OK, so my first thought was no, simply because this dependency is always available in the CLR at runtime. And so we don't expect to be subject to any of the usual breaking issues that warrant meticulous documentation of new dependencies.

I'm glad you cited this, though, because this assembly is very free of explicit references and this one was pulled in simply to generate a single test pattern (which I've now replaced with a string constant).